iam/api-client/service_account_keys.py

```diff
@@ -23,6 +23,7 @@
 
 import argparse
 # [START iam_create_key]
+# import base64
```

Comment:
Could you add a comment on uncommenting this when running the code, or move it down further where the rest of the base64 items with the `TODO` are commented out? It can be easily glanced over and users could easily get confused on why it's commented out.

Reply:
That's a great suggestion! Makes sense. 
Thank you Dan.